### PR TITLE
RaidenService with channelstate

### PR DESCRIPTION
**Pull request by hackaugusto**

### CODE REVIEW

**Comment by palango** [comment]
Fixes some issues. Good to go from my side.

File: raiden/raiden_service.py [hunk 1]
```diff
@@ -1255,14 +1255,14 @@ def direct_transfer_async2(self, token_address, amount, target, identifier):
         the target. After the message is sent the target is effectively paid
         and then it is not possible to revert.
 
-        The async result will be set to False iff there is no direct channel
+        The async result will be set to False if there is no direct channel
```
Comment (hackaugusto) [inline raiden/raiden_service.py]: https://en.wikipedia.org/wiki/If_and_only_if

Comment (palango) [reply raiden/raiden_service.py]: Ignorant me 😒

Comment (hackaugusto) [reply raiden/raiden_service.py]: nah, It's hard to tell when I get words right

File: raiden/raiden_service.py [hunk 2]
```diff
@@ -107,6 +194,8 @@ def load_snapshot(serialization_file):
         with open(serialization_file, 'rb') as handler:
             return pickle.load(handler)
 
+    return None
```
Comment (LefterisJP) [inline raiden/raiden_service.py]: Good! Being explicit on `None` as return value is really important imo.

**Comment by LefterisJP** [comment]
Note: I changed the `udp_message_handler.py` file to have the function defs in one line since they fit the 99 cols limit. As per our style guide if they fit in one line let's keep them so.

**Pull request merged**